src/components/diagrams/diagram-common.js

```diff
@@ -55,39 +55,18 @@ export const commonSldStyle = (theme, customStyle) => {
             {
                 stroke: theme.palette.text.primary,
             },
-
         '& .sld-flash, .sld-lock': {
             stroke: 'none',
             fill: theme.palette.text.primary,
         },
-        overflow: 'hidden',
-        ...customStyle,
-    };
-};
-
-export const commonNadStyle = (theme, customStyle) => {
-    return {
-        '& svg': {
-            // necessary because the default (inline-block) adds vertical space
-            // to our otherwise pixel accurate computations (this makes a
-            // scrollbar appear in fullscreen mode)
-            display: 'block',
-            width: '100%',
-            height: '100%',
-        },
-        '& .nad-label-box': {
-            color: theme.palette.text.primary,
-            'font-family': theme.typography.fontFamily,
-        },
-        '& .nad-text-edges': {
-            stroke: theme.palette.text.primary,
+        '& .arrow': {
+            fill: theme.palette.text.primary,
         },
         overflow: 'hidden',
-        ...customStyle,
     };
 };
 
-export const commonDiagramStyle = (theme, customStyle) => {
+export const commonDiagramStyle = (theme) => {
```

Comment:
Maybe we should move `paperBorders` CSS class because it's used only in diagram.js and position-diagram for Paper component, and `divInvalid` is used in *Content components.